src/main/java/org/jabref/logic/bibtex/LatexFieldFormatter.java

```diff
@@ -47,16 +42,40 @@ public static LatexFieldFormatter buildIgnoreHashes(LatexFieldFormatterPreferenc
         return new LatexFieldFormatter(true, prefs);
     }
 
+    private static void checkBraces(String text) throws InvalidFieldValueException {
+        List<Integer> left = new ArrayList<>(5);
+        List<Integer> right = new ArrayList<>(5);
+        int current = -1;
+
+        // First we collect all occurrences:
+        while ((current = text.indexOf('{', current + 1)) != -1) {
+            left.add(current);
+        }
+        while ((current = text.indexOf('}', current + 1)) != -1) {
+            right.add(current);
+        }
+
+        // Then we throw an exception if the error criteria are met.
+        if (!right.isEmpty() && left.isEmpty()) {
+            throw new InvalidFieldValueException("'}' character ends string prematurely.");
```

Comment:
As this gets shown to the user (or at least I remember having seen such message when I left the field). wouldn't it be better to describe the message a bit more simpler?